prow/plugins/jira/jira.go

```diff
@@ -220,10 +220,11 @@ func insertLinksIntoLine(line string, issueNames []string, jiraBaseURL string) s
 	return line
 }
 
-// replaceStringIfHasntSquareBracketOrSlashPrefix replaces a string if it is not prefixed by
-// a `[` which we use as heuristic for "Already replaced" or a `/` which we use as heuristic
-// for "Part of a link in a previous replacement".
-// It golang would support backreferences in regex replacements, this would have been a lot
+// replaceStringIfHasntSquareBracketOrSlashPrefix replaces a string if it is not prefixed by:
+// * `[` which we use as heuristic for "Already replaced",
+// * `/` which we use as heuristic for "Part of a link in a previous replacement",
+// * ``` (backtick) which we use as heuristic for "Inline code".
+// If golang would support back-references in regex replacements, this would have been a lot
 // simpler.
 func replaceStringIfHasntSquareBracketOrSlashPrefix(text, old, new string) string {
```

Comment:
Should we change the function name accordingly?
I am not even sure of it: it is already very long.
Naming is hard to me.

@alvaroaleman WDYT?

Reply:
Maybe ``replaceStringIfAppropriate`` or just ``linkifyLine``
Those can provide more info on the purpose than the actual implementation

Reply:
Done!